packages/react-app/src/views/BuilderListView.jsx

```diff
@@ -82,6 +85,45 @@ const BuilderStatusCell = ({ status }) => {
   );
 };
 
+const secondsPerDay = 24 * 60 * 60;
+const BuilderStreamCell = ({ stream }) => {
+  if (!stream) return <Box>-</Box>;
+
+  const cap = ethers.BigNumber.from(stream.cap);
+  const frequency = stream.frequency;
+  const frequencyDays = frequency / secondsPerDay;
+  const vestedPercentage = (new Date().getTime() / 1000 - stream.last) / frequency;
+  const vestedAmount = cap.mul(Math.round(new Date().getTime() / 1000 - stream.last)).div(frequency);
+  const available = cap.lt(vestedAmount) ? cap : vestedAmount;
+
+  const balanceStr = ethers.utils.formatEther(stream.balance);
+  const capStr = ethers.utils.formatEther(cap);
+  const availableStr = ethers.utils.formatEther(available);
+  return (
+    <Box>
```

Comment:
I think we are showing all the data we need.

You can work on the display if you want. This is how I see it rn:

![1](https://user-images.githubusercontent.com/2486142/157009038-fd004bde-2fdd-4a94-9f90-3b34d48f6c7e.png)


We want it to be closer to the current one in BG:

![2](https://user-images.githubusercontent.com/2486142/157009087-ed8436e4-8574-4414-8d73-5b0900c751dd.png)